src/cockpit/channel.py

```diff
@@ -389,7 +390,10 @@ def do_resume_send(self) -> None:
 
     def do_open(self, options):
         self.receive_queue = asyncio.Queue()
-        asyncio.create_task(self.run_wrapper(options), name=f'{self.__class__.__name__}.run_wrapper({options})')
+        if sys.version_info >= (3, 8):
+            asyncio.create_task(self.run_wrapper(options), name=f'{self.__class__.__name__}.run_wrapper({options})')
+        else:
+            asyncio.create_task(self.run_wrapper(options))
```

Comment:
I don't think cockpit-askpass uses that, so this could also get a polyfill wrapper.

Reply:
I tried that, but that's a bit difficult as the function does not exist at all in 3.6. Let's rather fix the systemd_ctypes regression and then bump it to latest main.